graphql/resolvers.js

```diff
@@ -236,12 +236,70 @@ const getPostById = async ({ id }, req) => {
   return responseData;
 };
 
+const updatePost = async ({ id, postInput }, req) => {
+  console.log({ 'update-post-response': postInput });
+  //  Check if user not authenticated
+  if (!req.isAuth) {
+    const error = new Error('Not authenticated');
+    error.code = 401;
+    throw error;
+  }
+
+  // Get post
+  const getPost = await Post.findById(id).populate('creator');
+
+  if (!getPost) {
+    const error = new Error('No post found');
+    error.code = 404;
+    throw error;
+  }
+
+  // Check user doesn't authorize
+  if (getPost.creator._id.toString() !== req.userId.toString()) {
+    const error = new Error('Not authorized');
+    error.code = 403;
+    throw error;
+  }
+
+  //  Validate input email
+  const errors = [];
+
+  if (errors.length > 0) {
+    const error = new Error('Invalid input.');
+    error.data = errors;
+    error.code = 422;
+    throw error;
+  }
```

Comment:
Validation

Incorrect comment. `// Validate input email` we doesn't email validate